modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java

```diff
@@ -769,7 +771,9 @@ else if (type.equals(DDMFormFieldTypeConstants.TEXT)) {
 				document.addGeoLocation(
 					name.concat("_geolocation"), latitude, longitude);
 			}
-			else if (type.equals(DDMFormFieldTypeConstants.SELECT)) {
+			else if (type.equals(DDMFormFieldTypeConstants.SELECT) ||
```

Comment:
Same thing as above.

Reply:
please, do [this](https://github.com/liferay-bpm/liferay-portal/pull/5277#discussion_r2446247293).